buildSrc/src/main/java/org/elasticsearch/gradle/LoggedExec.java

```diff
@@ -66,7 +65,6 @@ public void execute(Task task) {
         }
     }
 
-    @Internal
```

Comment:
that slipped through in the extra sourceSet before as gradles `validateProperties` task didn't catch it. by moving this to the main sourceSet now we need to fix it here to have :buildSrc:validateProperties pass